CHANGELOG.md

```diff
@@ -1,16 +1,157 @@
 ## UNRELEASED
 
+FEATURES:
+* Connect: Support transparent proxy. [[GH-905](https://github.com/hashicorp/consul-helm/pull/905)]
+  This feature allows users to reach other services on the Consul Service Mesh by using KubeDNS instead of using `localhost` and enforces all inbound and outbound traffic within a pod to go through the Envoy proxy.
+  Please see [Transparent Proxy](https://www.consul.io/docs/connect/transparent-proxy ) docs for more information.
+
+  **Note: This feature requires consul-k8s `v0.26.0` or higher.**
```

Comment:
```suggestion
  **Note: This feature requires Consul 1.10.0 or higher and consul-k8s `v0.26.0` or higher.**
```

Reply:
Do we have separate docs for tproxy we can link to as well?

Reply:
Line 6 seems to link to the consul.io docs to tproxy..do you mean other docs?

Reply:
gotcha, nvm